src/main/java/org/synyx/urlaubsverwaltung/person/web/PersonNotificationsMapper.java

```diff
@@ -70,134 +72,40 @@ static List<MailNotification> mapToMailNotifications(PersonNotificationsDto pers
 
     static PersonNotificationsDto mapToPersonNotificationsDto(Person person) {
 
-        final boolean isBossOrOffice = person.hasRole(Role.BOSS) || person.hasRole(Role.OFFICE);
-        final boolean isBossOrDHOrSSA = person.hasRole(Role.BOSS) || person.hasRole(Role.DEPARTMENT_HEAD) || person.hasRole(Role.SECOND_STAGE_AUTHORITY);
-        final boolean isOfficeOrBossOrDHOrSSA = person.hasRole(Role.OFFICE) || isBossOrDHOrSSA;
-
         final List<MailNotification> activePersonMailNotifications = new ArrayList<>(person.getNotifications());
-
         final PersonNotificationsDto personNotificationsDto = new PersonNotificationsDto();
+
+        final Map<MailNotification, Consumer<PersonNotificationDto>> setterByNotification = new HashMap<>();
```

Comment:
hiermit verlieren wir die Hilfe des compilers. falls `MailNotification` einen neuen typ bekommt, gibt es keine Meldung mehr über den fehlenden `default` case beim switch case.

in zeile 105 wird über alle werte iteriert. zur Laufzeit bekommen wir es also mit, weil `map.get` `null` zurückgibt und auf `null` dann `accept` aufgerufen werden soll. das wird mit unit tests abgedeckt. von daher passt das imho. das uns der compiler nicht mehr warnt.